pom.xml

```diff
@@ -149,9 +187,13 @@
     </profiles>
 
     <properties>
+        <jcommander.version>1.72</jcommander.version>
+        <slf4j.version.range>[1.7.0,1.8.0)</slf4j.version.range>
+        <org.json.version>20180130</org.json.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <performance.apim.version>0.1.1-SNAPSHOT</performance.apim.version>
         <performance.common.version>0.2.1-SNAPSHOT</performance.common.version>
+        <jwt.generator.version>1.0.0</jwt.generator.version>
```

Comment:
Remove jwt.generator.version. We don't need a specific version for jwt-generator